### ISSUE TITLE
< and > symbols in HTML tags getting replaced with &lt; and &gt;

### ISSUE THREAD

**Issue by TsKalandadze** [siyuan-note/siyuan#13155]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Can the issue be reproduced with the default theme (daylight/midnight)?

- [X] I was able to reproduce the issue with the default theme

### Could the issue be due to extensions?

- [x] I've ruled out the possibility that the extension is causing the problem.

### Describe the problem

When adding HTML code via HTML block, it renders HTML content as expected, but if I add multiple HTML blocks together in a super block and than delete one of them.  "<" and ">" symbols in HTML are replaced with `&lt;` and `&gt;` (which are HTML symbols for  "<" and ">"). Therefore HTML content is not rendered properly any more, it is rendered as text. 

### Expected result

"<" and ">" symbols in HTML should not be replaced with `&lt;` and `&gt;`

### Screenshot or screen recording presentation

https://github.com/user-attachments/assets/505d7569-4a6c-45ad-9f71-4ae843d300a2

![2024-11-14_210133](https://github.com/user-attachments/assets/2007a561-5ef8-4ac0-b3f2-a3bdbd2de403)

### Version environment

```markdown
- Version: v3.1.11
- Operating System: Windows 11 22H2 (22621.4317)
- Browser (if used): No Browser Used
```

### Log file

[siyuan.log](https://github.com/user-attachments/files/17753714/siyuan.log)

**Comment by TsKalandadze** [comment]
I made a mistake "In Expected results"

Instead of: "<" and ">" symbols in HTML should not be replaced with < and >
It should be:  "<" and ">" symbols in HTML should not be replaced with &lt; and &gt;

**Comment by TCOTC** [comment]
This is another issue related to `<` and `>` that I hope will be resolved as well: #13135
